exercicios/para-casa/exercicio-feito-lista.py

```diff
@@ -0,0 +1,12 @@
+num = int(input("Digite um dígito de número: "))
+
+par = []
+impar = []
+
+for i in range(20):
+    if num % 2 == 0:
+        par.append(num)
+        print("Lista de números pares: ", par)
```

Comment:
Não considero errado fazer o print dentro do for, mas acredito que estaria mais de acordo com o enunciado imprimir cada lista somente uma vez, depois do laço for.